platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/internal/RichTextEditorStyleExt.kt

```diff
@@ -31,22 +31,22 @@ import io.element.android.wysiwyg.view.PillStyleConfig
 import io.element.android.wysiwyg.view.StyleConfig
 import kotlin.math.roundToInt
 
-internal fun RichTextEditorStyle.toStyleConfig(context: Context): StyleConfig = StyleConfig(
+fun RichTextEditorStyle.toStyleConfig(context: Context): StyleConfig = StyleConfig(
```

Comment:
Is it absolutely necessary to expose these library internals?

Reply:
Reverted this in https://github.com/matrix-org/matrix-rich-text-editor/pull/891/commits/3562b60032879d37b886eb953af0fbeeb0309d0e